src/nvim/ui.coffee

```diff
@@ -8,6 +8,43 @@ remote = require 'remote'
 config = require './config'
 {keystrokeForKeyboardEvent} = require './key_handler'
 
+# borrowed from underscore@acf826fa42ba6
+_ = {}
+
+_.now = Date.now or ->
+  (new Date).getTime()
+
+_.debounce = (func, wait, immediate) ->
```

Comment:
I'm OK if you want to keep the `debounce` function, please just remove `_` and move `debounce` right after `get_vim_button_name`